client/ayon_blender/plugins/load/load_blend_link.py

```diff
@@ -19,7 +19,13 @@
 class BlendLinkLoader(plugin.BlenderLoader):
     """Link assets from a .blend file."""
 
-    product_types = {"*"}
+    product_types = {
+        "model", "camera", "rig",
+        "action", "layout", "blendScene", 
+        "animation", "workfile", "audio", 
+        "usd", "render", "image", 
+        "plate"
+    }
```

Comment:
```suggestion
    product_types = {
        "model", "camera", "rig",
        "action", "layout", "blendScene", 
        "animation", "workfile"
    }
```
Sorry for monday's brain :(